boards/arm/ebyte_e73_tbb_nrf52832/ebyte_e73_tbb_nrf52832.dts

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright (c) 2017 Shawn Nock <shawn@monadnock.ca>
+ * Copyright (c) 2017 Linaro Limited
+ * Copyright (c) 2021 morsisko
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <nordic/nrf52832_qfaa.dtsi>
+
+/ {
+	model = "EBYTE E73-TBB NRF52832";
+	compatible = "nordic,ebyte-e73-tbb-nrf52832";
```

Comment:
Nordic is not a vendor of this board, so it should be rather `"ebyte,e73-tbb-nrf52832"`.